pkg/k8s/apis/cilium.io/v1alpha1/version.go

```diff
@@ -7,4 +7,4 @@ package v1alpha1
 // Used to determine if CRD needs to be updated in cluster
 //
 // Developers: Bump patch for each change in the CRD schema.
-const CustomResourceDefinitionSchemaVersion = "1.1.8"
+const CustomResourceDefinitionSchemaVersion = "1.1.9"
```

Comment:
Please also remove that change in the case where you remove any changes from `types.go`.

Reply:
Done